client/my-sites/controller.js

```diff
@@ -137,7 +137,7 @@ function renderNoVisibleSites( context ) {
 		action: i18n.translate( 'Change Visibility' ),
 		actionURL: '//dashboard.wordpress.com/wp-admin/index.php?page=my-blogs',
 		secondaryAction: i18n.translate( 'Create New Site' ),
-		secondaryActionURL: `${ signup_url }?ref=calypso-nosites`,
+		secondaryActionURL: `${ getOnboardingUrl() }?ref=calypso-nosites`,
```

Comment:
Did we mean to call this without a `state` arg?

Reply:
Nop, my bad 🤦  
Fixed in https://github.com/Automattic/wp-calypso/pull/48350/commits/ee9761c3130cce82926a739153bc4495a50eff90